src/services/svgUtils.js

```diff
@@ -184,6 +184,8 @@ function getRootDefs () {
     if (!rootSVG) {
       rootSVG = document.createElementNS(SVGNS, 'svg')
       rootSVG.setAttribute('id', 'RandomArtRootSVG')
+      rootSVG.style.position = 'absolute'
```

Comment:
@nicksellen I sometimes had a big white space at the bottom, this fixed it. Should I leave it in here or is there a better solution?

Reply:
Ah I see, I didn't realise it would have taken up any space in the DOM. If it fixes it then fine!